src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,83 @@
+package edu.kis.vh.nursery;
+
+/**
+ * The type Default counting out rhymer.
+ */
+public class DefaultCountingOutRhymer {
+
+    /**
+     * The constant INT.
+     */
+    public static final int INT = 12;
+    /**
+     * The constant INT1.
+     */
+    public static final int INT1 = -1;
+    /**
+     * The constant INT2.
+     */
+    public static final int INT2 = 11;
+    private int[] numbers = new int[INT];
+
+    private int total = INT1;
+
+    /**
+     * Gets total.
+     *
+     * @return the total
+     */
+    public int getTotal() {
+        return total;
+    }
+
+    /**
+     * Count in.
+     *
+     * @param in the in
+     */
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    /**
+     * Call check boolean.
+     *
+     * @return the boolean
+     */
+    public boolean callCheck() {
+        return total == INT1;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.